oracle.py

```diff
@@ -351,6 +356,46 @@ def check_workload_log(workload_log):
             bug_report += "[ERROR][WORKLOAD] %s" % line
     return alarm, bug_report
 
+BORING_EVENT_OBJECT_FIELDS = ["managedFields", "annotations", "labels"]
+BORING_EVENT_OBJECT_PATHS = [
+                            "data",
+                            "metadata/resourceVersion",
+                            "metadata/generateName",
+                            "metadata/ownerReferences",
+                            "spec/containers/0/image",
```

Comment:
@laphets Since you are ignoring `image` in the first container, you might also want to do it for other containers. I am not sure whether the lib you are using supports `spec/containers/*/image` or not. We can live with it for this PR if it does not cause false alarms.